platform/android/app/src/main/kotlin/coop/polypoly/polypod/FeatureFragment.kt

```diff
@@ -261,7 +261,8 @@ open class FeatureFragment : Fragment() {
                 }
             }
         }
-        return ExternalFile(url = url, name = name, size = size)
+        return if (size> 0)
```

Comment:
Nit: I'm not sure why ktlint tolerates it, but I suppose it'd be nice to change `(size> 0)` to `(size > 0)`.